cshared/auth_method_cshared.go

```diff
@@ -0,0 +1,192 @@
+// +build ignore
+package main
+
+import (
+	"C"
+	"reflect"
+	"strings"
+
+	"golang.org/x/crypto/ssh"
+	"gopkg.in/src-d/go-git.v3/clients/http"
+	. "gopkg.in/src-d/go-git.v3/clients/ssh"
```

Comment:
Can you remove the dot import, is a bad practice in Go -> https://github.com/golang/go/wiki/CodeReviewComments#import-dot